src/engine/console.cpp

```diff
@@ -521,62 +594,97 @@ bool consolekey(int code, bool isdown)
                 break;
 
             case SDLK_RIGHT:
-                if(commandpos>=0 && ++commandpos>=(int)strlen(commandbuf)) commandpos = -1;
+                if (commandpos >= 0 && ++commandpos >= (int)strlen(commandbuf)) 
+                {
+                    commandpos = -1;
+                }
                 break;
 
             case SDLK_UP:
-                if(histpos > history.length()) histpos = history.length();
-                if(histpos > 0) history[--histpos]->restore();
+                if (histpos > history.length())
+                {
+                    histpos = history.length();
+                }
+                if (histpos > 0) 
+                {
+                    history[--histpos]->restore();
+                }
                 break;
 
             case SDLK_DOWN:
-                if(histpos + 1 < history.length()) history[++histpos]->restore();
+                if (histpos + 1 < history.length())
+                {
+                    history[++histpos]->restore();
+                } 
                 break;
 
             case SDLK_TAB:
-                if(commandflags&CF_COMPLETE)
+                if (commandflags & CF_COMPLETE)
                 {
                     complete(commandbuf, BIGSTRLEN, commandflags&CF_EXECUTE ? "/" : NULL);
-                    if(commandpos>=0 && commandpos>=(int)strlen(commandbuf)) commandpos = -1;
+                    if(commandpos >= 0 && commandpos >= (int)strlen(commandbuf)) commandpos = -1;
```

Comment:
Either use brackets all the time, or don't for one liners. Stick to one style.

Reply:
same as before, I stopped making the code follow our style guidelines to avoid confusion.